src/main/kotlin/io/hsar/wh40k/combatsimulator/logic/actionoptions/AttackActionOption.kt

```diff
@@ -0,0 +1,57 @@
+package io.hsar.wh40k.combatsimulator.logic.actionoptions
+
+import io.hsar.wh40k.combatsimulator.model.UnitInstance
+import io.hsar.wh40k.combatsimulator.model.unit.Attribute
+import io.hsar.wh40k.combatsimulator.model.unit.BodyPart
+import io.hsar.wh40k.combatsimulator.model.unit.NumericValue
+import io.hsar.wh40k.combatsimulator.random.AverageDice
+import io.hsar.wh40k.combatsimulator.random.RandomDice
+import io.hsar.wh40k.combatsimulator.random.RollResult
+
+abstract class AttackActionOption: ActionOption() {
+    /*
+    Provide some concrete implementations of common damage calculation tasks that can be re-used by child classes
```

Comment:
This probably ought to be above the class declaration, not under it.

Reply:
Updated